__tests__/app.test.js

```diff
@@ -49,7 +49,8 @@ describe('/api/articles/:article_id', () => {
                     body: expect.any(String),
                     created_at: expect.any(String),
                     votes: expect.any(Number),
-                    article_img_url: expect.any(String)
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number),
```

Comment:
Really nice but this should be added in a new separate test as it is a new behaviour being added.